src/webhooks.ts

```diff
@@ -113,54 +78,42 @@ function generateSignature(webhookSecretKey: string) {
     submissionId: Object
     formId: string
     epoch: number
-  }) {
+  }) => {
+    if (!this.secretKey) {
```

Comment:
Can we move this check to the constructor?

Reply:
As discussed, left here for flexibility on function calls. The `authenticate` function does not need a secret key to be called.